### PR TITLE
chore: enable `prettier/prettier` rule for `eslint`

### PR DESCRIPTION
**Pull request by faustbrian**
`utils` was using `6.8.0` while all others used `8.0.0` and that is what caused `utils` to blow up with `TypeError: Error while loading rule 'prettier/prettier': context.getPhysicalFilename is not a function` because the `prettier` plugin assumes `eslint@8.0.0` or higher. eslint now lives at the root package.json so this is no longer an issue.

### CODE REVIEW

**Comment by gabrocheleau** [comment]
Shouldn't we move that dep to the root package.json like you've done in #2163 ?

**Review by gabrocheleau — APPROVED** [review]

**Review by acolytec3 — APPROVED** [review]
Making linting great again, LGTM!

**Comment by holgerd77** [comment]
It's really hard to follow up with what's going on in this repository. 😜

Phew. But great stuff everywhere. Nice. 🔥
